src/components/remote_control/src/resource_allocation_manager_impl.cc

```diff
@@ -70,31 +87,70 @@ AcquireResult::eType ResourceAllocationManagerImpl::AcquireResource(
 }
 
 void ResourceAllocationManagerImpl::SetAccessMode(
-    AccessMode::eType access_mode) {
+    const AccessMode::eType access_mode) {
   current_access_mode_ = access_mode;
 }
 
 void ResourceAllocationManagerImpl::AskDriver(const std::string& module_type,
-                                              uint32_t app_id,
+                                              const uint32_t hmi_app_id,
                                               AskDriverCallBackPtr callback) {
   LOG4CXX_AUTO_TRACE(logger_);
-  // Send GetInteriorConsent
-  // subscribe on GetInteriorConsent response
-  // execute callback on response
-  // event_dispatcher_.add_observer(function_id, msg[kId].asInt(), callback);
+  // Create GetInteriorConsent request to HMI
+  Json::Value params;
+  params[json_keys::kAppId] = hmi_app_id;
```

Comment:
@LuxoftAKutsan I thought about it but could see no sense as this message as created & used only here. Thus I decided to leave it here on stack.